pkg/infra/aws/aws.go

```diff
@@ -13,3 +13,270 @@
 // limitations under the License.
 
 package aws
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws"
+	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ec2"
+	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/eks"
+	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/iam"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func EKSDeploy(ctx *pulumi.Context) error {
+	//	conf := config.New(ctx, "")
+	//	ec2Size := conf.Get("ec2Size")
+
+	vpc, err := ec2.NewVpc(ctx, "mo-pulumi-vpc", &ec2.VpcArgs{
+		CidrBlock:          pulumi.String("10.0.0.0/16"),
+		EnableDnsHostnames: pulumi.Bool(true),
+		EnableDnsSupport:   pulumi.Bool(true),
+		Tags: pulumi.StringMap{
+			"Name": pulumi.String("mo-pulumi-vpc"),
+		},
+	})
+	if err != nil {
+		return err
+	}
+	ctx.Export("VPC_ID", vpc.ID())
+
+	azState := "available"
+	zoneList, err := aws.GetAvailabilityZones(ctx, &aws.GetAvailabilityZonesArgs{
```

Comment:
under one region or not?

Reply:
one region configed by aws config (like cn-northwest-1)

Reply:
> under one region or not?

one region configed by aws config (like cn-northwest-1)